File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.py
def solution(a, b, n):
    answer = 0
    while n >= a:
        coke = (n // a) * b  # 얻을 수 있는 콜라의 갯수
        answer += coke
        n = n % a + coke  # 남은 빈 병에 새로 받은 콜라의 빈 병을 더해줌
    return answer

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b, int n) {
    int answer = 0;
    
    while(n >= a){
        int coke = (n / a) * b; // 새로 받은 콜라
        answer += coke;
        n = (n % a) + coke;
    }
    
    return answer;
}